src/handlers/graphql.ts

```diff
@@ -1,11 +1,11 @@
 import { ApolloServer, BaseContext } from '@apollo/server';
 import { startServerAndCreateLambdaHandler } from '@as-integrations/aws-lambda';
 import { GraphQLError } from 'graphql';
-import { Maybe } from '../generated/graphql';
+import { Maybe } from '@generated/graphql';
 import { resolvers, typeDefs } from '../schema';
```

Comment:
Does this work with `@schema`? Or `@schema/index`?

Reply:
oh! yes, I missed this one